src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,116 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "abcdefg";
+    private static final String VALID_PASSWORD = "123456";
+    private static final String LOGIN_LESS_THAN_6_CHARACTERS = "abcde";
+    private static final String PASSWORD_LESS_THAN_6_CHARACTERS = "12345";
+    private static final int AGE_LESS_THAN_MIN = 10;
+    private static final int MIN_AGE = 18;
+    private static RegistrationService registrationService;
+    private User user = new User(VALID_LOGIN, VALID_PASSWORD, MIN_AGE);
```

Comment:
You should initialize it in @BeforeAll method

Reply:
@BlueVioletTeti according to common mistakes file, we have to "move all hardcoded values to constant fields and give them informative names" and this is what I did :)